cmd/initContainer/main.go

```diff
@@ -247,20 +262,19 @@ func removePolicyReport(client *client.Client, kind string) error {
 
 	// name of namespace policy report follows the name convention
 	// pr-ns-<namespace name>
+	var wg sync.WaitGroup
+	wg.Add(len(namespaces.Items))
 	for _, ns := range namespaces.Items {
-		reportNames := []string{
-			fmt.Sprintf("policyreport-ns-%s", ns.GetName()),
-			fmt.Sprintf("pr-ns-%s", ns.GetName()),
-			fmt.Sprintf("polr-ns-%s", ns.GetName()),
-		}
-
-		var wg sync.WaitGroup
-		wg.Add(len(reportNames))
-		for _, reportName := range reportNames {
-			go deleteResource(client, "", kind, ns.GetName(), reportName, &wg)
-		}
-		wg.Wait()
+		go func(namespaceName string) {
```

Comment:
Do we still want to spin up goroutines? Not sure if it would improve runtime.

Reply:
I will try and test that with Kind to see if there is any improvement.